packages/target-native-android-emulator/src/create-android-emulator-target.js

```diff
@@ -3,15 +3,15 @@ const tempy = require('tempy');
 const osnap = require('osnap/src/android');
 const { createWebsocketTarget } = require('@loki/target-native-core');
 
-const captureScreenshot = async () => {
+const captureScreenshot = async (device) => {
```

Comment:
Hmm, I wonder if this approach actually works with concurrency. Wouldn’t the story change on all simulators at once? How else would you target it? Do you build and run three separate servers on different ports?

Reply:
Yeah, our setup runs 3 seperate packagers, instances of storybook and loki all running different randomised ports.

Reply:
Okie 👍